src/memray/commands/summary.py

```diff
@@ -26,6 +27,30 @@ def prepare_parser(self, parser: argparse.ArgumentParser) -> None:
             type=int,
             default=None,
         )
+        alloc_type_group = parser.add_mutually_exclusive_group()
+        alloc_type_group.add_argument(
+            "--temporary-allocation-threshold",
+            help=dedent(
+                """\
+            Report temporary allocations, as opposed to leaked allocations or
+            high watermark allocations.  An allocation is considered temporary
+            if at most N other allocations occur before it is deallocated.  With
+            N=0, an allocation is temporary only if it is immediately
+            deallocated before any other allocation occurs.
+            """
+            ),
+            action="store",
+            dest="temporary_allocations",
+            type=int,
+            default=0,
```

Comment:
@godlygeek I recalled you wanted this to be n-1 so the normal version will run with --temporary-allocation-threshold=1, no? What value should we use to flag that's not activated?

Reply:
> What value should we use to flag that's not activated?

I'll go with -1 for now, but if you don't like that we should be able to use `None` instead.